integration/inspektor-gadget/advise_networkpolicy_test.go

```diff
@@ -44,8 +44,9 @@ func TestAdviseNetworkpolicy(t *testing.T) {
 			Cmd: fmt.Sprintf(`$KUBECTL_GADGET advise network-policy monitor -n %s --output ./networktrace-client.log &
 					sleep 10
 					kill $!
-					head networktrace-client.log | sort | uniq`, nsClient),
-			ExpectedRegexp: fmt.Sprintf(`{"node":".*","namespace":"%s","pod":"test-pod","container":"test-pod","timestamp":.*,"type":"normal","netnsid":.*,"pktType":"OUTGOING","proto":"tcp","port":9090,"podHostIP":".*","podIP":".*","podLabels":{"run":"test-pod"},"remoteKind":"svc","remoteAddr":".*","remoteName":"test-pod","remoteNamespace":"%s","remoteLabels":{"run":"test-pod"}}`, nsClient, nsServer),
```

Comment:
I always wanted to update these tests not to use regexp. I did that in https://github.com/inspektor-gadget/inspektor-gadget/commit/d3956b7c8e8339866ff215bea7f94eb93f177e0b. Feel free to pick up that, otherwise I can open a PR with that one too.

Reply:
Thanks! Please open a separate PR.